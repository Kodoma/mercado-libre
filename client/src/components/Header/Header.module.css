.header {
    background-color: #fff159;
    border: 0;
    position: relative;
    height: 100px;
    width: 100%;
}

.logo {
    background-image: url(https://http2.mlstatic.com/frontend-assets/ui-navigation/5.14.5/mercadolibre/logo__large_plus.png);
    height: 34px;
    top: 30px;
    width: 134px;
    background-repeat: no-repeat;
    overflow: hidden;
    text-indent: -999px;
    position: absolute;
    left: 5%;
}

@media (max-width: 812px) {
    .logo {
        background-image: url(https://http2.mlstatic.com/frontend-assets/ui-navigation/5.14.5/mercadolibre/logo__small@2x.png);
        background-size: 44px 31px;
        width: 44px;
        height: 32px;
        top: 24px;
        left: 10px;
    }
}
