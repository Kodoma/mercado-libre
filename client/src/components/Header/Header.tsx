import React from "react";
import styles from "./Header.module.css";
import Search from "./Search/Search";


const Header = () => {
    return(
        <header className={styles.header}>
            <a className={styles.logo} href="//www.mercadolibre.com.ar">
                Mercado Libre Argentina - Donde comprar y vender de todo
            </a>
            <Search />
        </header>
    )
}

export default Header