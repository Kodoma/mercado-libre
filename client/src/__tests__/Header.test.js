import React from 'react';
import { shallow } from 'enzyme';
import Header from "../components/Header/Header";

describe("tests Header", () => {
    it("should render my Header component", () => {
        const component = shallow(<Header />);
        expect(component.getElements()).toMatchSnapshot();
    });
});