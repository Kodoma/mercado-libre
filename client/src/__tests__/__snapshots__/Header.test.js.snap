// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests Header should render my Header component 1`] = `
Array [
  <header
    className="header"
  >
    <a
      className="logo"
      href="//www.mercadolibre.com.ar"
    >
      Mercado Libre Argentina - Donde comprar y vender de todo
    </a>
    <SearchAutocomplete />
  </header>,
]
`;
